index.js

```diff
@@ -89,22 +90,24 @@ module.exports = {
       srcDir: '/',
       destDir: 'velocity'
     });
+    velocityTree = map(velocityTree, 'velocity/velocity.js', function(content) {
+      return 'if (typeof FastBoot === \'undefined\') { ' + content + ' }';
+    });
 
     var matchMediaPath = path.dirname(require.resolve('match-media'));
     var matchMediaTree = new Funnel(this.treeGenerator(matchMediaPath), {
       srcDir: '/',
       destDir: 'match-media'
     });
+    matchMediaTree = map(matchMediaTree, 'match-media/matchMedia.js', function(content) {
+      return 'if (typeof FastBoot === \'undefined\') { ' + content + ' }';
+    });
 
     return mergeTrees([tree, velocityTree, matchMediaTree]);
   },
 
-  included: function(app){
-    if (process.env.EMBER_CLI_FASTBOOT) {
-      // in fastboot we use the shim by itself, which will make
-      // importing velocity a noop.
-      this.import('vendor/shims/velocity.js');
```

Comment:
Don't we need to make sure this is imported too (only in the fastboot world)? This can use the `appendFastBootManifest` API.